### PR TITLE
Feature/ppxlink

### PR DESCRIPTION
**Pull request by timosachsenberg**
Just to sync with develop

### CODE REVIEW

**Comment by timosachsenberg** [comment]
@enetz merge?

**Comment by cbielow** [comment]
could `void SpectraIdentificationViewWidget::addTextItemToBottomRow_` also be templated on the value?! Would reduce code duplication...

File: src/openms_gui/source/VISUAL/SpectraIdentificationViewWidget.cpp [hunk 1]
```diff
@@ -533,75 +458,234 @@ namespace OpenMS
...
-      // zoom scan
-      item = table_widget_->itemPrototype()->clone();
-      if ((*layer_->getPeakData())[i].getInstrumentSettings().getZoomScan())
-      {
-        item->setText("yes");
-      }
-      else
-      {
-        item->setText("no");
+          if (IDFilter().getBestHit(current_pi, false, best_ph))
```
Comment (cbielow) [inline src/openms_gui/source/VISUAL/SpectraIdentificationViewWidget.cpp]: ouch.
`getBestHit()` is static....

Comment (timosachsenberg) [inline src/openms_gui/source/VISUAL/SpectraIdentificationViewWidget.cpp]: k will fix

**Pull request merged**